src/unittest/test_translations.cpp

```diff
@@ -25,12 +25,41 @@ TEST_CASE("test translations")
 {
 	SECTION("Plural-Forms function for translations")
 	{
-		auto form = GettextPluralForm::parseHeaderLine(L"Plural-Forms: nplurals=3; plural= (n-1+1)<=1 ? n||1?0:1 : 1?(!!2):2;");
-		REQUIRE(form);
-		REQUIRE(form->size() == 3);
+#define REQUIRE_FORM_SIZE(x) {REQUIRE(form); REQUIRE(form->size() == (x));}
```

Comment:
use `REQUIRE(form && form->size() == N)`?

Reply:
I get this error when I apply this change:
```
/home/y5nw/Documents/minetest/lib/catch2/catch_amalgamated.hpp:5547:27: error: static assertion failed due to requirement 'always_false<bool>::value': operator&& is not supported inside assertions, wrap the expression inside parentheses, or decompose it
 5547 |             static_assert(always_false<RhsT>::value,
      |                           ^~~~~~~~~~~~~~~~~~~~~~~~~
/home/y5nw/Documents/minetest/src/unittest/test_translations.cpp:31:3: note: in instantiation of function template specialization 'Catch::operator&&<bool>' requested here
   31 |                 REQUIRE_FORM_SIZE(2);
      |                 ^
/home/y5nw/Documents/minetest/src/unittest/test_translations.cpp:28:46: note: expanded from macro 'REQUIRE_FORM_SIZE'
   28 | #define REQUIRE_FORM_SIZE(x) {REQUIRE((form) && (form->size() == (x)));}
      |                                              ^
1 error generated.
```

Aside from that, I prefer the current one because it allows telling the exact problem if the `REQUIRE` fails: you can tell whether it parsed the `nplurals` part incorrectly or the `plural` expression.